Vagrantfile.dist

```diff
@@ -30,10 +30,10 @@ class CustomConfig
     parameters_file = ENV.fetch('VIRTUALIZATION_PARAMETERS_FILE', 'virtualization/parameters.yml')
     config = YAML::load(File.open(parameters_file))
 
-    @lxc_box_url = config['lxc_box_url'] || "http://vagrantbox-public.liip.ch/liip-jessie64-lxc.box"
-    @lxc_box_name = config['lxc_box_name'] || 'jessie64-lxc'
-    @vbox_box_url = config['vbox_box_url'] || 'http://vagrantbox-public.liip.ch/liip-jessie64.box'
-    @vbox_box_name = config['vbox_box_name'] || 'jessie64'
+    @lxc_box_url = config['lxc_box_url']     || config['box_url']  || 'https://vagrantbox-public.liip.ch/drifter-jessie64-base.json'
+    @lxc_box_name = config['lxc_box_name']   || config['box_name'] || 'drifter/jessie64-base'
+    @vbox_box_url = config['vbox_box_url']   || config['box_url']  || 'https://vagrantbox-public.liip.ch/drifter-jessie64-base.json'
+    @vbox_box_name = config['vbox_box_name'] || config['box_name'] || 'drifter/jessie64-base'
```

Comment:
I don't remember exactly why we did it like that but if I am not mistaken if redundant with the `custom_config.get` method which also has a default now.

Also, with the new `json`  format for the boxes, I think we could drop all that, release v2, and inform about the BC :P

But let's do that in another PR or never ^^